Секция состоит из двух подсекций footer и newsletter

Amindlog opened the pull request.

Amindlog committed:
enter footer and newsletter

Amindlog committed:
буду в носить изменения

Amindlog committed:
uikit-NewslitterSection добавил

Amindlog committed:
Uikit footer newletter

Amindlog committed:
Merge branch 'develop' into footer

nobleworkshop commented on src/pages/uikit.jsx:
```diff
@@ -11,6 +11,8 @@ import UiKitCopyright from "../uikit-components/uikit-Copyright";
 import UiKitRunningLine from "../uikit-components/uikit-RunningLine";
 import UiKitMostPopular from "../uikit-components/uikit-MostPopular";
+import UiKitFooter from "../uikit-components/uikit-Footer";
+import UiKitNewsletter from "../uikit-components/uikit-Newsletter";
```
UiKitNewsletter уже подключен выше на 9-й строке из того же файла. Нет ли тут путаницы с другим компонентом Newsletter который был в виджетах справа?

nobleworkshop reviewed (changes requested):
В текущем состоянии проект из ветки Footer не запускается. Ошибки и на что ругается оставлю здесь и в комментариях к коду.

1) В файле `src/pages/uikit.jsx`
9 строка:
`import UiKitPrintmagazine from "../uikit-components/uikit-Printmagazine";`
при этом данный компонент (файл) отсутствует.


Записал видео. Отправлю в чат.

Стоит пофиксить ошибки. 

Оформить весь подвал в компонент Footer, уже внутри которого будет внетренний блок FooterNewsLetter и другие если это необходимо.

Commit:
Фикс. Удалил подключение компонентов которые уже переименованы

Amindlog committed:
правлю....

nobleworkshop reviewed (changes requested):
Предыдущие правки не внесены, и здесь писал и в чат отправлял видео.

Amindlog committed:
footer1.0

Amindlog committed:
footer финальная версия

Amindlog commented:
В нес изменения

nobleworkshop reviewed (changes requested):
По композиции, то что весь компонент теперь в Footer - хорошо.

Правки:

1) Проверить и сделать соответствие pixel perfect согласно дизайну.

2) Черный фон в footer должен идти на всю ширину страницы. Сейчас не так.

3) Компоненты которые используются только в Footer предлагаю назвать с приставкой Footer. (Как сделалано с компонентами которые в виджетах в сайдбаре)

Amindlog committed:
Исправил верстку, более мение подогнал через pixel

Amindlog commented:
> По композиции, то что весь компонент теперь в Footer - хорошо.
> 
> Правки:
> 
> 1. Проверить и сделать соответствие pixel perfect согласно дизайну.
> 2. Черный фон в footer должен идти на всю ширину страницы. Сейчас не так.
> 3. Компоненты которые используются только в Footer предлагаю назвать с приставкой Footer. (Как сделалано с компонентами которые в виджетах в сайдбаре)

исправил

Amindlog committed:
Исправил uikit-footer

Commit:
Fix footer

Поправил подвал:
- Разбил на компоненты
- Общий контейнер
- Переработал стили и классы, обертки

nobleworkshop reviewed (approved):
Внёс правки, доработал.

nobleworkshop committed:
Merge branch 'develop' into footer
